gr-digital/lib/costas_loop_cc_impl.cc

```diff
@@ -126,8 +122,8 @@ float costas_loop_cc_impl::phase_detector_2(gr_complex sample) const
 
 float costas_loop_cc_impl::phase_detector_snr_8(gr_complex sample) const
 {
-    float K = (sqrt(2.0) - 1);
-    float snr = abs(sample) * abs(sample) / d_noise;
+    const float K = (sqrt(2.0) - 1);
+    const float snr = abs(sample) * abs(sample) / d_noise;
```

Comment:
|z|·|z| = z·conj(z); please replace by 

`const float snr = sample * std::conj(sample) / d_noise;` 

for superior compute.